bin/local-env/install-composer.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-COMPOSER_VERSION=`curl -Ls -w %{url_effective} -o /dev/null https://github.com/composer/composer/releases/latest | rev | cut -d '/' -f 1 | rev`
+COMPOSER_VERSION='1.10.1'
```

Comment:
```suggestion
COMPOSER_VERSION='1.10.17'
```
Change composer version.